src/i_video.c

```diff
@@ -799,6 +804,9 @@ void I_FinishUpdate(void)
             {
                 I_Sleep((remaining_time - 1000) / 1000);
             }
+#ifdef HAVE_PAUSE
+            _mm_pause();
```

Comment:
Wait a minute, thus needs a comment. What is happening here and how does it improve frame limiting?